app/assets/javascripts/libs/globals.js

```diff
@@ -0,0 +1,9 @@
+window.$ = window.jQuery = require('jquery');
+window.I18n = require('i18n-js');
+window._ = require('underscore');
```

Comment:
Do we still need these globals after this webpack config?
https://github.com/Codeminer42/cm42-central/pull/29/files#diff-a58d55bdb5770c78ad512f8e91f8d051R64